# moving obstacle points 
#point tl and br in rectangle 
points rect_tl
points rect_br
#optical flow points number
int32 point_size
#optical flow points
points[] points
